mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -372,7 +372,12 @@ Specifying a network enables private IP.
 At least `ipv4_enabled` must be enabled or a `private_network` must be configured.
 This setting can be updated, but it cannot be removed after it is set.
 
-* `require_ssl` - (Optional) Whether SSL connections over IP are enforced or not.
+* `require_ssl` - (Optional) Whether SSL connections over IP are enforced or not. Please also populate `ssl_mode`.
```

Comment:
Could you add the updates here in the docs to the field descriptions as well?

Also, the use of "please" in a few places here is actually a little unclear. I think we should disambiguate as either a requirement or a recommendation.